easybuild/easyconfigs/c/Clair3/Clair3-1.0.4-foss-2022a.eb

```diff
@@ -39,21 +39,25 @@ dependencies = [
     ('zstd', '1.5.2'),
     ('HTSlib', '1.15.1'),
     ('libdeflate', '1.10'),
+    ('PyPy', '7.3.12', '-3.10', SYSTEM),
```

Comment:
I spent quite a bit of time in trying to get PyPy to build from source rather than using pre-built binaries, but I didn't manage to get it fully working yet (and I gave up for now), see #18848